README.md

```diff
@@ -189,6 +189,20 @@ expect { EventsNotifier.with(profile: profile).canceled(event).notify_later }
   .to have_enqueued_notification(identify: "123", body: "Alarma!")
 ```
 
+Abstract Notifier provides two convinient minitest assertions:
+
+```ruby
+require 'abstract_notifier/testing/minitest'
```

Comment:
We should extend the core Minitest assertions; users MUST include our assertions module themselves. For example:

```ruby
require 'abstract_notifier/testing/minitest'

class NotifierTestCase < Minitest::Test
  include AbstractNotifier::TestHelper
end
```